requirements/libraries.txt

```diff
@@ -9,7 +9,7 @@ boto3==1.7.80
 # does until we can solve that problem.
 cryptography==2.9.2
 dj-database-url==0.5.0
-djangorestframework==3.9.1
+djangorestframework>=3.9.1,<4.0
```

Comment:
I'm sorry to keep doing this — but what's the reason for the version range here? Can we pin a definite version?

Reply:
Looks like Wagtail 2.10 requires DRF > 3.11.1, which looks like the latest release. Can we use that?

Reply:
I will test with djangorestframework==3.11.1

Reply:
I will test with djangorestframework==3.11.1